<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardWidget</class>
 <widget class="QWidget" name="KeyboardWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>865</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="10" colspan="2">
     <widget class="QPushButton" name="buttonf">
      <property name="text">
       <string>f</string>
      </property>
     </widget>
    </item>
    <item row="1" column="13" colspan="2">
     <widget class="QPushButton" name="buttony">
      <property name="text">
       <string>y</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2" colspan="2">
     <widget class="QPushButton" name="buttonq">
      <property name="text">
       <string>q</string>
      </property>
     </widget>
    </item>
    <item row="3" column="11" colspan="2">
     <widget class="QPushButton" name="buttonb">
      <property name="text">
       <string>b</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7" colspan="2">
     <widget class="QPushButton" name="buttonc">
      <property name="text">
       <string>c</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4" colspan="3">
     <widget class="QPushButton" name="button2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3" colspan="2">
     <widget class="QPushButton" name="buttonz">
      <property name="text">
       <string>z</string>
      </property>
     </widget>
    </item>
    <item row="3" column="18" colspan="2">
     <widget class="QPushButton" name="buttonDot">
      <property name="text">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="1" column="11" colspan="2">
     <widget class="QPushButton" name="buttont">
      <property name="text">
       <string>t</string>
      </property>
     </widget>
    </item>
    <item row="2" column="19" colspan="2">
     <widget class="QPushButton" name="buttonk">
      <property name="text">
       <string>k</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7" colspan="2">
     <widget class="QPushButton" name="button3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="20" colspan="2">
     <widget class="QPushButton" name="buttonSlash">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QPushButton" name="button1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8" colspan="2">
     <widget class="QPushButton" name="buttond">
      <property name="text">
       <string>d</string>
      </property>
     </widget>
    </item>
    <item row="0" column="27" colspan="3">
     <widget class="QPushButton" name="buttonEq">
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="1" column="24" colspan="3">
     <widget class="QPushButton" name="buttonSqBrOp">
      <property name="text">
       <string>[</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5" colspan="3">
     <widget class="QPushButton" name="buttons">
      <property name="text">
       <string>s</string>
      </property>
     </widget>
    </item>
    <item row="0" column="15" colspan="3">
     <widget class="QPushButton" name="button7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="21" colspan="2">
     <widget class="QPushButton" name="buttonl">
      <property name="text">
       <string>l</string>
      </property>
     </widget>
    </item>
    <item row="3" column="15" colspan="2">
     <widget class="QPushButton" name="buttonm">
      <property name="text">
       <string>m</string>
      </property>
     </widget>
    </item>
    <item row="2" column="12" colspan="2">
     <widget class="QPushButton" name="buttong">
      <property name="text">
       <string>g</string>
      </property>
     </widget>
    </item>
    <item row="1" column="30" colspan="2">
     <widget class="QPushButton" name="buttonBackSlash">
      <property name="text">
       <string>\</string>
      </property>
     </widget>
    </item>
    <item row="2" column="23" colspan="2">
     <widget class="QPushButton" name="buttonDots">
      <property name="text">
       <string>:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="17" colspan="2">
     <widget class="QPushButton" name="buttonj">
      <property name="text">
       <string>j</string>
      </property>
     </widget>
    </item>
    <item row="2" column="25" colspan="2">
     <widget class="QPushButton" name="buttonBr">
      <property name="text">
       <string>'</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QPushButton" name="buttonSt">
      <property name="text">
       <string>`</string>
      </property>
     </widget>
    </item>
    <item row="0" column="9" colspan="2">
     <widget class="QPushButton" name="button4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="13" colspan="2">
     <widget class="QPushButton" name="button6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="1" column="22" colspan="2">
     <widget class="QPushButton" name="buttonp">
      <property name="text">
       <string>p</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="3">
     <widget class="QPushButton" name="buttonw">
      <property name="text">
       <string>w</string>
      </property>
     </widget>
    </item>
    <item row="0" column="18" colspan="2">
     <widget class="QPushButton" name="button8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="20" colspan="2">
     <widget class="QPushButton" name="buttono">
      <property name="text">
       <string>o</string>
      </property>
     </widget>
    </item>
    <item row="1" column="7" colspan="2">
     <widget class="QPushButton" name="buttone">
      <property name="text">
       <string>e</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="22" colspan="2">
     <widget class="QPushButton" name="button0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="17">
     <widget class="QPushButton" name="buttonComma">
      <property name="text">
       <string>,</string>
      </property>
     </widget>
    </item>
    <item row="2" column="14" colspan="3">
     <widget class="QPushButton" name="buttonh">
      <property name="text">
       <string>h</string>
      </property>
     </widget>
    </item>
    <item row="1" column="18" colspan="2">
     <widget class="QPushButton" name="buttoni">
      <property name="text">
       <string>i</string>
      </property>
     </widget>
    </item>
    <item row="0" column="24" colspan="3">
     <widget class="QPushButton" name="buttonMin">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="15" colspan="3">
     <widget class="QPushButton" name="buttonu">
      <property name="text">
       <string>u</string>
      </property>
     </widget>
    </item>
    <item row="0" column="11" colspan="2">
     <widget class="QPushButton" name="button5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="13" colspan="2">
     <widget class="QPushButton" name="buttonn">
      <property name="text">
       <string>n</string>
      </property>
     </widget>
    </item>
    <item row="0" column="20" colspan="2">
     <widget class="QPushButton" name="button9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="4" column="7" colspan="10">
     <widget class="QPushButton" name="buttonSpace">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="9" colspan="2">
     <widget class="QPushButton" name="buttonv">
      <property name="text">
       <string>v</string>
      </property>
     </widget>
    </item>
    <item row="1" column="27" colspan="3">
     <widget class="QPushButton" name="buttonSqBrCl">
      <property name="text">
       <string>]</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QPushButton" name="buttonx">
      <property name="text">
       <string>x</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QPushButton" name="buttona">
      <property name="text">
       <string>a</string>
      </property>
     </widget>
    </item>
    <item row="1" column="9" colspan="2">
     <widget class="QPushButton" name="buttonr">
      <property name="text">
       <string>r</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
